imagekitio/file.py

```diff
@@ -83,6 +85,8 @@ def upload(
             raise TypeError(ERRORS.MISSING_UPLOAD_FILENAME_PARAMETER.value)
         url = "%s%s" % (URL.UPLOAD_BASE_URL, "/api/v1/files/upload")
         headers = self.request.create_headers()
+        if isinstance(file, io.BufferedReader):
+            file = (file_name, file, "image/jpeg")
```

Comment:
The content-type should be detected from the input file

While trying to upload a PDF file, the payload sent in MultiPart Request is `image/jpeg`, because it is hard coded

![Screenshot 2023-04-18 at 2 21 47 PM](https://user-images.githubusercontent.com/8640877/232725974-a7967e2c-d2ee-4e0e-bde6-15106b455d5d.png)